x-pack/plugins/secops/public/components/drag_and_drop/helpers.ts

```diff
@@ -76,6 +76,10 @@ interface AddProviderToTimelineParams {
     id: string;
     provider: DataProvider;
   }>;
+  addAndProvider?: ActionCreator<{
+    id: string;
+    provider: DataProvider;
+  }>;
```

Comment:
This doesn't look it's being used anywhere. Is this for future implementation or maybe I'm missing where it's dispatched at?

Reply:
oh yes correct, forget to delete it